flake.nix

```diff
@@ -35,13 +35,14 @@
           ...
         }:
         let
+          xcharter = pkgs.callPackage ./xcharter.nix { inherit pkgs; };
```

Comment:
Note that `callPackage` automatically passes attributes from `pkgs` to the given function.